recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: b28e72ac7a98be3d28ad28570299a393dfcd32e5e3f6a353dec94675767b6319
 
 build:
-  number: 2
-  skip: true  # [py2k]
+  number: 3
+  noarch: python
   script: '{{ PYTHON }} -m pip install . -vv '
 
 requirements:
```

Comment:
In `run` should be `python >=3.5`, see https://github.com/Delgan/loguru/blob/f31e97142adc1156693a26ecaf47208d3765a6e3/setup.py#L72